pkg/lib/cockpit-components-terminal.tsx

```diff
@@ -347,18 +436,16 @@
         this.setState(this.calculateDimensions());
     }
 
-    setTerminalTheme(theme) {
+    setTerminalTheme(theme: TerminalTheme) {
         this.terminal.options.theme = themes[theme];
     }
 
-    onBeforeUnload(event) {
+    onBeforeUnload(event: Event) {
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21680-85366158-20250829-093922-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)